### DISCUSSION TITLE
MUSIC LICENSING - Public execution licence for existing sound recording in live Comedy and magic show (canadian MRO category) (SOCAN 11B / Re:Sound 5.I)

### DISCUSSION THREAD

**Discussion by Liverace** [pair-cg/general-contributions#75] [User Stories]
# 1. Need:
_What is your need?_
This User Storie is drafted with the intent to describe the information need of Canadian copyright one-stop-shop outfit **Entandem** to be able to issue a licence to music users playing recorded music in public during a _Comedy and magic_ show. The user storie also encompasses the need to identify the sound recordings heard in the venue before, during and after the performance and trace its different rightsholders. 

Such information is often incomplete and publicly opened databases could perhaps help the identification process. Until recently (2012), the need to precisely identify the works, recordings and rightsholders had not been a prerequisite to issue a licence and usage remuneration sums had not been split granularly with regards to the specific efforts involved by the different rightsholders. This user storie will take into account the fact that such granular information and action is now becoming a need. 

**Entandem** is a one-stop-shop joint venture service created by Re:Sound and SOCAN acting on behalf of all Canadian Music Rights Organisation (MRO) sharing the representation of rightsholders involved in the creation and production of the existing sound recordings heard during the performance ([https://www.entandemlicensing.com/portal/en/dashboard](https://www.entandemlicensing.com/portal/en/dashboard)). These rightsholders segments and associated split shares are further described in section 2, 5 and 6.
 
# 2. Action:
_What is an action that can be taken towards fulfilling that need?_
Rightsholders of existing sound recordings are of three types and all have their respective MRO's.

The **Sound RECORDING Neighbouring right** Component 
1. Registered **producer** of the sound recording is represented by Musical Rights Organization [SOPROQ](https://www.soproq.org/) and/or [CONNECT](https://connectmusic.ca/Default.aspx).
2. The **artists** whose performances are fixed in the sound recording are represented by Musical Rights Organization [ARTISTI](https://www.artisti.ca/) and/or [CONNECT](https://connectmusic.ca/Default.aspx).
Additional intermediary Musical Rights Organization Re:Sound ([Re:Sound](https://www.resound.ca/)) agregates neighbouring rights component representation into the joint venture **Entandem** in partnership with SOCAN (seen next bullet).

The underlying **Musical WORK Copyright** Component 

3. Musical Rights Organization [SOCAN](http://www.socan.com/) represents both the **creators** and **publishers** of the underlying original work or work adaptation. 

![image](https://user-images.githubusercontent.com/3813232/112763002-a5a5ad00-8fd0-11eb-8f3d-cd3a354047d3.png)

To identify the sound recording performed, the following information is needed : 
- Performer Name
- Track Title (Sound recording title)
- _International Standard Recording Code_ [ISRC](https://www.wikidata.org/wiki/Property:P1243)

To trace the different rightsholders the following information is needed : 
- Sound recordings list must be supplied to Producer, Performer, Creator and Publisher MRO's via the one-stop-shop **Entandem**.
- A matching process has to be performer by **Entandem** or its MRO partners in order to attach the recordings ISRC to underlying works usually  identified using the _International Standard Work Code_ [ISWC](https://www.wikidata.org/wiki/Q949026). Missing ISRC in the recording's list, rightsholders or titles homonyms sometimes makes this work difficult or impossible.
- **Producer** identity can theoretically be infered by MRO's using the ISRC since this identifier is composed of a country prefix and producer prefix. Despite this, some dysfunctions can occur (duplicate ISRC, inadequate country identification, etc.).
- **Performers** identity is theoretically attached to ISRC in MRO's databases. Performers all have an [IPN](https://www.scapr.org/tools-projects/ipd/), a private and closed identifier used by the performer's MRO. 
- **Creators** and **Publishers** all have an IPI, a semi-private identifier used by copyright MRO's ([IPI](https://www.wikidata.org/wiki/Q19832975)). 
- Two openly public unique identifiers have been created to try to mitigate the private nature and difficult access of the IPI and ISWC : The _International Standard Name Identifier_ (for moral and physical persons) [ISNI](https://www.wikidata.org/wiki/Q423048) and the very recent _Best Open Work Identifier_ [BOWI](https://bowi.io/). 

After an event, Promoter must curently register and follow the following steps with **Entandem** > New Report > Submit Report 
Report must contain : 
Event Name
Venue
Performer Name
Promoter Name
Promoter Address
Province, City, Postal Code
Date of Event
 
# 3. Desired Outcome:
_What is the desired outcome?_
Ease identifing sound recordings played during performances and trace rightsholders for these sound recordings.
**Current filed report information isn't sufficient to allow accurate and granular identification of works, recordings and rightsholders.** An efficient recording, work and rightsholders matching process calls for the triangulation of a maximum of data sources, but until now, this effort hasn't been supported by open and linked data. 
 
# 4. Perspective:
_From what perspective is this user story coming from?_
One-stop-shop **Entandem** (SOCAN-Re:Sound)
 
# 5. Stakeholder Group:
_Which stakeholder group most closely fits the perspective of this User Story?_
Individual music creators and their music publishers (Musical WORK Component)
Individual (performing) artists, and producers (Sound RECORDING Component)
both trough their joint venture **Entandem**.
 
# 6. Additional info:
_Here's more information._
The [Copyright Board of Canada](https://cb-cda.gc.ca/en) certifies the tariffs to be paid.
Sound RECORDING Component Collective Management Organization splits revenues equally between (performing) artists and producers. 
Musical WORK Component Collective Management Organization splits revenues according to agreements between creators and publishers 
Information travels between Entandem, SOCAN, Re:Sound, SOPROQ, CONNECT and Artisti using proprietary EDI format or [DDEX-MLC](https://kb.ddex.net/display/MLC11/MLC+Home) choreography (to be investigated).

**Comment by Liverace** [comment]
Hello @fjjulien I will need feedback!

**Reply by fjjulien** [reply]
As discussed, I think this user story would be most effective if written primarily from the perspective of Entandem. 
For actions, in addition to the reporting process, we should document the analysis/research to identify the sound recording performed, and to trace the different rightsholders for this sound recording.

**Reply by fjjulien** [reply]
The distribution actions can also be briefly eluded to. But I think all information needs can be identified just with the reporting and analysis/tracing processes.

**Reply by fjjulien** [reply]
The main desired outcome should be "Easily identify sound recordings played during preformances and trace rightsholders for these sound recordings." Payment processing doesn't need to be mentioned in the desired outcomes.

**Reply by mariel-marshall** [reply]
Hi @Liverace thanks for this contribution. I think @fjjulien is right and that a bit of clarity is need around what perspective you are writing from. Are you writing from the perspective of a technology creator like Entandem? I also got a bit confused about what your need is. Is it that you need to know what sound recordings are played during a specific preformance? Perhaps you could walk us through things step by step.

**Comment by fjjulien** [comment]
Salut @Liverace. In the "Need" section, you seem to suggest that rights traceability information is needed for both sound recordings played during the actual performance (incidental use of music) as well as for music played before and after the performance (background use of music). As far as I know, SOCAN only does direct distribution for the sound recording used as part of the performance (the incidental use of music). Entandem consequently only seeks recording information for this incidental use of music. For this particular need, one key action for Entandem is to conduct an enquiry to identify, with limited information on the event name and on the performer name, what particular sound recordings were played as part of the performance. I presume this must involve inquiring directly with the performer/producer, because this kind of information isn't available anywhere (as your "desired outcome" suggests).

**Reply by Liverace** [reply]
Entandem must require adequate information at reporting stage from the licensee. SOCAN have extensive databases for internal usage and therefore can match such report to both Work and Recording rights holders. I believe "real life" testing and case study could be performed to better document the current workflow, flaws and desired improvements.

**Reply by fjjulien** [reply]
Should we invite someone from Entandem and/or SOCAN to review and comment on this user story? I'd be happy to ask them.

**Reply by Liverace** [reply]
After trying to emulate an event and a reporting procedure which I did'nt complete, Michel Giguère, an External Licensing Agent at Entandem has written me to enquire about any difficulty experienced on the portal. I could contact him.

**Comment by fjjulien** [comment]
Note - There are other live performance tariffs for the which the licensing form requires additional administrative information:

- SOCAN 11.A/Re:Sound 5.E (Circus, Ice Shows, Fireworks): gross receipts from ticket sales
- Re:Sound 5.K (Recorded music accompanying live entertainment): total admissions
- Re:Sound 5.J (Concerts): total capacity. In this particular context, “capacity” means the maximum number of persons that can attend an event (seating and standing) based on the number of tickets that can be issued for the event (both free and paid) as reported in the event’s ticket manifest or other relevant document.
